INSTALL.md

```diff
@@ -132,9 +30,7 @@ git branch
 The highlighted branch is the one that is currently used. Let's assume it's `shadowrobot_1234` for the following instructions. On the newly installed computer you will need to pull the same branch:
 
 ```bash
-roscd sr_ethercat_hand_config
-git pull
-git fetch shadowrobot_12345
+curl -L bit.ly/dev-install | bash -s -- -w ~/projects/shadow_robot/base -c shadowrobot_1234
```

Comment:
working on fixing that bit.ly (it was pointing to the branch, I'm trying to point it to master now)